### ISSUE TITLE
Implement Deadman function

### ISSUE THREAD

**Issue by nathanielc** [influxdata/flux#1634]
The deadman has been designed in  #1338. Implement the function as defined there.

**Comment by jlapacik** [comment]
`deadman` is defined as:
```
deadman = (d, tables=<-) => tables
    |> max(column: "_time")
    |> map(fn: (r) => ( {r with dead: r._time < now() - d} ))
```

Unfortunately the expression `now() - d` will not type check until the type system supports polymorphic literals (assuming durations are representable as time values). Otherwise we will need to define a new binary operator for performing time value addition (this would be similar to OCaml's `+` and `+.` operators for integer and floating point addition respectively).

**Comment by pauldix** [comment]
This doesn't look like it's taking into account the discussion on: https://github.com/influxdata/flux/pull/1588#pullrequestreview-268731198

**Comment by pauldix** [comment]
Closed by #1650
